MvvmCross.Plugins/Color/Plugin.cs

```diff
@@ -3,21 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using MvvmCross.Converters;
-using MvvmCross.Plugins;
 
 namespace MvvmCross.Plugin.Color
 {
+    [MvxPlugin]
     [Preserve(AllMembers = true)]
-    public class PluginLoader
-        : IMvxPluginLoader
+    public sealed class PluginLoader : IMvxPlugin
```

Comment:
This class isn't a platform implementation and it has a `MvxPlugin` attribute. Is that correct? Also seems like the class should be renamed to `Plugin` maybe?

Reply:
Yes, this is correct. The PluginLoader class was pretty much useless, so we are just registering plugins instead